topsort-api-v2.yml

```diff
@@ -466,6 +515,230 @@ components:
           description: >
             Human-readable explanation of or details about the error.
             The string for a given error may change over time; code should not parse or dispatch based on particular values for this field.
+    Placement:
+      type: object
+      required:
+        - path
+      properties:
```

Comment:
I think we are missing `product_id`, `category_id` and `query`. This is what wicha asked